yt_dlp/extractor/lsm.py

```diff
@@ -208,7 +210,7 @@ def _real_extract(self, url):
             r'window\.ltvEmbedPayload\s*=', webpage, 'embed json', video_id)
         embed_type = traverse_obj(data, ('source', 'name', {str}))
 
-        if embed_type == 'telia':
+        if embed_type == 'backscreen':
```

Comment:
assuming it's not harmful, could we do this
```suggestion
        if embed_type in ('backscreen', 'telia'):  # 'telia' only for backwards compat
```